blogs/explainable_ai/AI_Explanations_on_CAIP.ipynb

```diff
@@ -1122,41 +1105,6 @@
       ],
       "execution_count": 0,
       "outputs": []
-    },
-    {
-      "cell_type": "code",
-      "metadata": {
-        "id": "mXERenKAjfdK",
-        "colab_type": "code",
-        "colab": {}
-      },
-      "source": [
-        "!rm taxi-data.json\n",
-        "taxi_data_json = valid_data.iloc[7:27].to_dict(orient='records')\n",
-        "with open('taxi-data.json', 'a') as outfile:\n",
-        "  for instance in taxi_data_json:\n",
-        "    json.dump(instance, outfile)\n",
-        "    outfile.write('\\n')\n",
-        "\n",
-        "resp_obj = !gcloud beta ai-platform explain --model $MODEL --version $VERSION_IG --json-instances='taxi-data.json'\n",
-        "response_IG = json.loads(resp_obj.s)\n",
-        "\n",
-        "resp_obj = !gcloud beta ai-platform explain --model $MODEL --version $VERSION_SHAP --json-instances='taxi-data.json'\n",
-        "response_SHAP = json.loads(resp_obj.s)\n",
-        "\n",
-        "with open('IG_attributions.json', 'a') as outfile:\n",
-        "  for x in range(20):\n",
-        "    json.dump(response_IG['explanations'][x]['attributions_by_label'][0]['attributions'], outfile)\n",
-        "    outfile.write('\\n')\n",
-        "\n",
-        "with open('SHAP_attributions.json', 'a') as outfile:\n",
-        "  for x in range(20):\n",
-        "    json.dump(response_SHAP['explanations'][x]['attributions_by_label'][0]['attributions'], outfile)\n",
-        "    outfile.write('\\n')\n",
-        "\n"
-      ],
-      "execution_count": 0,
-      "outputs": []
     }
```

Comment:
This cell was just used to generate data for the blog post and isn't needed for the example.